### DISCUSSION TITLE
Problem connecting to Tinymovr through CAN Bus using CANine firmware

### DISCUSSION THREAD

**Discussion by SpeakersOFF** [tinymovr/Tinymovr#311] [Q&A - Tinymovr]
Hello, 
I'm having a problem connecting to the tiny move. Someone help?
----

$ lsusb
Bus 002 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 001 Device 008: ID 0483:c1b0 **STMicroelectronics CANine CDC**
Bus 001 Device 004: ID 0e0f:0008 VMware, Inc. Virtual Bluetooth Adapter
Bus 001 Device 003: ID 0e0f:0002 VMware, Inc. Virtual USB Hub
Bus 001 Device 002: ID 0e0f:0003 VMware, Inc. Virtual Mouse
Bus 001 Device 001: ID 1d6b:0001 Linux Foundation 1.1 root hub
~$ sudo tinymovr_cli
Tinymovr Studio 1.6.2
Listening for nodes...

In [1]: 

it does seem to find it.

**Comment by yconst** [comment]
Hi, have you tried setting the TERM DIP switch to on?

**Reply by SpeakersOFF** [reply]
I have tried both on and off, but it still doesn't work. I bought two more boards and two can controllers. I still have the same issue. Just for setup I have 24vdc wire to +,- and the can controller is connected to an can 1 on the motor controller

**Reply by yconst** [reply]
Is this a virtual machine?

**Reply by SpeakersOFF** [reply]
YES.

**Reply by yconst** [reply]
Could you provide an image of your setup, with clearly shown connections?

In addition, have you tried using slcan instead of the CANine firmware (just as a verification)? 

I have never used the setup in a virtual machine. Even though it should not be an issue since it is discovering your interface, I'd try on a physical machine just for good measure.
